services/src/main/webapp/WEB-INF/web.xml

```diff
@@ -21,18 +21,37 @@
 
     <servlet-mapping>
         <servlet-name>FileServlet</servlet-name>
-        <url-pattern>/staticReport/*</url-pattern>
+        <url-pattern>/static-report/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>org.jboss.windup.web.services.rest.RestApplication</servlet-name>
+        <url-pattern>/*</url-pattern>
+    </servlet-mapping>
+
+    <servlet-mapping>
+        <servlet-name>org.jboss.windup.web.services.rest.RestApplicationFurnace</servlet-name>
+        <url-pattern>/graph/*</url-pattern>
     </servlet-mapping>
 
     <security-constraint>
         <web-resource-collection>
             <web-resource-name>secured</web-resource-name>
-            <url-pattern>/rest/*</url-pattern>
+            <url-pattern>/*</url-pattern>
         </web-resource-collection>
         <auth-constraint>
             <role-name>user</role-name>
         </auth-constraint>
     </security-constraint>
+    <security-constraint>
+        <!-- TODO: I don't think static reports should be public,
+             but we have to use different authentication for it
```

Comment:
Because of the way how we approach static reports. For any API calls, we already have auth token stored in JavaScript. But we open static reports in new browser window and we don't have tokens available. 

We should have it either somewhere in cookies (I would prefer that) or pass it in URL.